x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/upgraded_cluster/100_geo_shape_doc_values.yml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
@iverase I also am investigating errors with this file. I am experiencing inconsistent behavior and investigating why, but I think most of what is tested here is good. would love to know what you think!

Reply:
I think the issue here is that we are force merging on the case of mixed cluster?

Reply:
I found out what the problem is.

I was writing these tests assuming they were only run on 7.8 upgrading to 8.0, but we also have these tests run with 8.0 doing a rolling-restart to 8.0. I've updated the tests with skips, but will need to sync with core-infra to see what they recommend. I am a bit surprised